kafka-rest/src/main/java/io/confluent/kafkarest/entities/v3/CreateTopicRequest.java

```diff
@@ -72,6 +80,9 @@ static CreateTopicRequest fromJson(
 
     public abstract Builder setReplicationFactor(@Nullable Short replicationFactor);
 
+    @SuppressWarnings("checkstyle:LineLength")
+    public abstract Builder setReplicasAssignments(@Nullable Map<Integer, List<Integer>> replicasAssignments);
```

Comment:
Break the line on params start:

```
public abstract Builder setReplicasAssignments(
    @Nullable Map<Integer, List<Integer>> replicasAssignments);
```

We have a strict 100-cols policy in REST Proxy.